modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_request_decisions_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class PowerOfAttorneyRequestDecisionsController < ApplicationController
+      before_action :set_poa_request, only: :create
+
+      ##
+      # TODO: We ought to centralize our exception rendering. Starting here for
+      # now.
+      #
+      concerning :ExceptionRendering do
```

Comment:
It looks like this concern shows up in your PR #13 also. Can we get this abstracted out to make the code reusable please?

Reply:
@LindseySaari This gets extracted in [#13](https://github.com/department-of-veterans-affairs/vets-api/pull/20057/files#diff-081964771dc985241e37a67c3547f6134cf51f67f9b598df41be785512f02e19).

Reply:
Ok sounds great! Thanks!